dashboards/postgres/main.json

```
@@ -0,0 +1,400 @@
+{
+  "metric_keys": [
+    "vector_pg_up"
+  ],
+  "dashboard": {
+    "title": "PostgreSQL",
+    "description": "",
+    "visuals": [
+      {
+        "title": "pg_stat_database_deadlocks_total",
+        "description": "Number of deadlocks detected in this database.",
```

I wonder if it would be preferable to give the graphs names that are regular-human-meaningful? (as opposed to PostgreSQL-database-administrator-meaningful)

I do think it's a good idea to reference the native PostgreSQL name for the metric, as it provides something to Google to know more.

In a way, I guess I'm suggesting swapping the title and the description? Perhaps making "Deadlocks" the title and "per database (pg_stat_database_deadlocks_total)" the description?

(Remember that the UI puts the title and the description _in the same line_, which limits the space available for both -- hence my shorter suggestion)

```suggestion
        "title": "Deadlocks",
        "description": "per database and host (pg_stat_database_deadlocks_total)",
```

(Same thing would apply for other metrics if you agree)

---

We shouldn't show the metric name in the graph title, no. Useful for us debugging to see which metrics are working, not so much as a user friendly graph title.